dev/Repeater/InspectingDataSource.h

```diff
@@ -24,6 +24,7 @@ class InspectingDataSource :
 #pragma endregion
 
     int IndexOf(winrt::IInspectable const& value);
+    int IndexOfCore(winrt::IInspectable const& value);
```

Comment:
move into region above and rename IdataSourceOverrides into ItemsSourceViewOverrides